docs/docs/segment-time.md

```diff
@@ -26,3 +26,12 @@ Show the current timestamp.
 - time_format: `string` - format to use, follows the [golang standard][format] - defaults to `15:04:05`
 
 [format]: https://gobyexample.com/time-formatting-parsing
+
+- template: `string` - A go [text/template][go-text-template] template extended with [sprig][sprig] utilizing the
+properties below. Only used when a value is set, making the above properties obsolete.
+
+  example: `{{ now | date \"January 02, 2006 (MST)\" | lower }}`
```

Comment:
I feel like the example in the docs should include time, since time is what people are likely using this segment for.  It will save them some time of having to look at the golang docs if we have something like this:

```md
example: `{{ now | date \"January 02, 2006 (MST) 15:04:05 PM\" | lower }}`
```